### ISSUE TITLE
Auto 'Fix Grey Characters' setting

### ISSUE THREAD

**Issue by Ebonmark** [BarleyFlour/WrathVisualAdjustmentsMod#36]
Not sure if it's feasible, but if there was a checkbox that could be ticked off to have the mod automatically 'Fix Grey Characters' on zone load in, that would be handy.

_Originally posted by @Ebonmark in https://github.com/BarleyFlour/WrathVisualAdjustmentsMod/issues/19#issuecomment-917545670_

**Comment by LeNawax** [comment]
Yes, this is a must-have considering that the bug appears everytime after a loading when the mod is installed.

**Comment by VanoraBlackheart** [comment]
Maybe an shortcut hotkey that you can just press in game without having to go crtl + 10 open the mod then reset. Would be a blessing. Its so annoying that each time you load a new zone or get attacked randomly boom your grey. >.<

**Comment by BarleyFlour** [comment]
Due to the nature of the problem automating it isn't possible, though i've added a hotkey (alt+x) for rebuilding all characters as of v1.10.2

**Comment by VanoraBlackheart** [comment]
Thank you very much for the hotkey. Appriciate all your work on this keep up the good work :D
